Графика для всех страниц

Serega560 opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2248995/tasks/5)

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2248995/tasks/7)

Serega560 committed:
Сделал разметку headera и начал разметку main

Serega560 committed:
Сделал разметку главной страницы

Serega560 committed:
Начал размечать страницу каталога

Serega560 committed:
разметил каталог

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+  <header class="page-header">
+    <h1 class="visiually-hidden">Магазин "Девайс"
+    </h1>
+      <a href="index.html"><img src="#" alt="Логотип Магазина Девайс" class="logo logo-header"></a>
```
тебе нужно соблюдать форматирование документа для соблюдения требований академии и удобочитаемости документа. Касается и главной страницы
В vscode есть автоформат, частично поможет исправить
On Windows Shift + Alt + F
On Mac Shift + Option + F
On Linux Ctrl + Shift + I

Serega560 replied:
Исправил и одну и другую страницы

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+    <h1 class="visiually-hidden">Магазин "Девайс"
+    </h1>
+      <a href="index.html"><img src="#" alt="Логотип Магазина Девайс" class="logo logo-header"></a>
+  <nav class="navigation">
```
в nav лучше вынести основные ссылки: каталог и тд

Serega560 replied:
вынес каталог, доставка, гарантия, контакты

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+      <a href="index.html"><img src="#" alt="Логотип Магазина Девайс" class="logo logo-header"></a>
+  <nav class="navigation">
+    <ul class="navigation-list">
+      <li class="navigation-item"><input type="text" placeholder="Поиск по сайту"></li>
+      <li class="navigation-item"><button type="serch">Найти</button></li>
```
это форма. input с label + кнопка type submit

Serega560 replied:
Обернул в форму

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+    <ul class="navigation-list">
+      <li class="navigation-item"><input type="text" placeholder="Поиск по сайту"></li>
+      <li class="navigation-item"><button type="serch">Найти</button></li>
+      <img src="" alt="Иконка человека">
```
декоративная графика в разметке не нужна

Serega560 replied:
убрал

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+      <li class="navigation-item"><button type="serch">Найти</button></li>
+      <img src="" alt="Иконка человека">
+      <li class="navigation-item"><a href="#" class="navigation-link">Войти</a></li>
+      <img src="" alt="Иконка сравнения">
```
img не нужна

Serega560 replied:
убрал

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+      <li class="navigation-item"><a href="#" class="navigation-link">Войти</a></li>
+      <img src="" alt="Иконка сравнения">
+      <li class="navigation-item"><a href="#" class="navigation-link">Сравнить</a></li>
+      <img src="" alt="Иконка Корзины">
```
img не нужна

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+      <li class="navigation-item"><a href="#" class="navigation-link">Корзина</a></li>
+    </ul>
+
+<p><a href="#">КАТАЛОГ ТОВАРОВ</a></p>
```
параграф тут не нужен

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+    </ul>
+
+<p><a href="#">КАТАЛОГ ТОВАРОВ</a></p>
+<button type="button"></button>
```
у кнопки должен быть тип и описание

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+  </li>
+  <li class="navigation-item"><a href="#" class="navigation-link">Контакты</a>
+</ul>
+</ul>
```
лишний закрывающий тег

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+  </nav>
+  </header>
+  <main class="main-inner">
+<header class="header-inner">
```
тебе не нужен тут еще один хедер

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+</head>
+<body>
+  <header class="page-header">
+    <h1 class="visiually-hidden">Магазин "Девайс"
```
основной заголовок тут будет "Моноподы для селфи". На странице должен быть один  заголовок h1

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+  <main class="main-inner">
+<header class="header-inner">
+  <h1 class="inner-header-title">Моноподы для селфи</h1>
+<ul class="breadcrumbs-list">
```
лучше ol

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+<header class="header-inner">
+  <h1 class="inner-header-title">Моноподы для селфи</h1>
+<ul class="breadcrumbs-list">
+  <li class="breadcrumbs-item"><a class="breadcrumbs-link" href="index.html"></a>Главная</li>
```
текст вне ссылки

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+  <h1 class="inner-header-title">Моноподы для селфи</h1>
+<ul class="breadcrumbs-list">
+  <li class="breadcrumbs-item"><a class="breadcrumbs-link" href="index.html"></a>Главная</li>
+  <li class="breadcrumbs-item"><a class="breadcrumbs-link" href="catalog.html">Каталог товаров</li>
+  <li class="breadcrumbs-item"> Моноподы для селфи</li>
+</ul>
```
невалидная разметка

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+  <li class="breadcrumbs-item"> Моноподы для селфи</li>
+</ul>
+
+<section>
```
правильнее будет выделить секцию каталога, а внутри нее выделитьЖ фильтры, сортировку, список товаров и пагинацию

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+<section>
+  <h2 class="filter">Фильтр</h2>
+  <form action="" class="catalog-filter" method="get"></form>
+<fieldset class="catalog-filter-group">
```
это внутри формы

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+<fieldset class="catalog-filter-group">
+
+  <legend class="catalog-filter-title">Цвет</legend>
+  <Ul class="catolog-filter-list">
```
теги с маленькой буквы все, это критерий написания кода, так разметка нерегистрозависима

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+
+  <legend class="catalog-filter-title">Цвет</legend>
+  <Ul class="catolog-filter-list">
+    <li class="catolog-filter-item"><label class="control">
+      <input type="checkbox" name="black" checked>Черный
+    </label></li>
+    <li class="catolog-filter-item"><label class="control">
+      <input type="checkbox" name="white" checked>Белый
+    </label></li>
+    <li class="catolog-filter-item"><label class="control">
+      <input type="checkbox" name="blue">Синий
+    </label></li>
+    <li class="catolog-filter-item"><label class="control">
+      <input type="checkbox" name="red">Красный
+    </label></li>
+    <li class="catolog-filter-item"><label class="control">
+      <input type="checkbox" name="pink">Розовый
+    </label></li>
+</Ul>
```
чекбоксы нужно сгрупировать по имени, но у них будет уникальный value, так же тут разметка получается невалидна из-за незакрытых ранее тегов

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+</label></li>
+</ul>
+</fieldset>
+ <button type="button">Показать</button>
```
тип кнопке. Форма должна закончиться здесь

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+
+<section class="select-control">
+  <h2>Сортировка</h2>
+  <option value="">Популярные</option>
```
тут заголовок тогда будет h3, сортировка просто select, внутри option

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+  <h2>Сортировка</h2>
+  <option value="">Популярные</option>
+</section>
+<a href="#" class="button button-up">
+  <span class="visually-hidden">Вверх</span>
+</a>
+<a href="#" class="button button-down">
+  <span class="visually-hidden">Вниз</span>
+</a>
```
это кнопки сортировки по убыванию, по возрастанию

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+  <li class="product-card">
+    <a href="#" class="product-card-link">
+      <img src="product-card-image" alt="">
+      <h3 class="product-card-title">Любительская селфи-палка</h3>
```
вероятно уровень изменится, будет видно когда выше исправишь

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+          </li>
+        </ul>
+
+        <a href="">Загрузить еще</a>
```
кнопка загрузки, перехода не будет
button type="button"

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+          </li>
+        </ul>
+
+</header>
```
тут закончился main

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+</header>
+  </main>
+  <footer class="page-footer">
+    <img src="" alt="Магазин Девайс" class="footer-contacts-logo">
```
это ссылка на главную

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+    <section class="footer-contacts">
+    <address class="cotacts-address">
+      <p class="contacts-address-text">Санкт-Петербург, набережная реки Карповки, 5, литера П.</p>
+        <a href="#" class="contacts-phone" >Тел.: +7 (812) 812-12-12</a>
```
в href будет tel

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+      <p class="contacts-address-text">Санкт-Петербург, набережная реки Карповки, 5, литера П.</p>
+        <a href="#" class="contacts-phone" >Тел.: +7 (812) 812-12-12</a>
+    </address>
+    <img src="№" alt="Желтая палочка">
```
img не нужна

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+    </section>
+
+    <section class="footer-social">
+    <a href=""><img src="#" alt="Теллеграмм"></a>
```
список соцсетей, без img

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+    <header class="page-header">
+      <h1 class="visiually-hidden">Магазин "Девайс"
+      </h1>
+        <img src="slider-1.png" alt="Логотип Магазина Девайс" class="logo logo-header">
```
можно оставить ссылку, но без href. Упростит переиспользование, далее аналогично каталогу

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+    <img src="#" alt="Квадрокоптер">
+    <p>Порхает как бабочка, жалит как пчела!</p>
+    <p>Этот обычный, на первый взгляд, квадрокоптер оснащен мощным лазером, замаскированным под стандартную камеру.</p>
+    <p><a href="#">Подробнее</a></p>
```
абзац не нужен

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+    </header>
+
+<main class="main-index">
+    <button type="button" class="visiually-hidden">Кнопка назад</button>
```
тут будет секция со слайдером, состоящая из 3-х слайдов

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+
+<main class="main-index">
+    <button type="button" class="visiually-hidden">Кнопка назад</button>
+    <button type="button" class="visiually-hidden">Кнопка вперед</button>
```
это кнопка смены фото, ей не нужен этот класс

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+    <p>Порхает как бабочка, жалит как пчела!</p>
+    <p>Этот обычный, на первый взгляд, квадрокоптер оснащен мощным лазером, замаскированным под стандартную камеру.</p>
+    <p><a href="#">Подробнее</a></p>
+    <p>800 м</p>
+    <p>Дальность полета</p>
+    <p>50 м</p>
+    <p>Радиус поражения</p>
```
лучше теги dl, dt,dd

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+    <p>Дальность полета</p>
+    <p>50 м</p>
+    <p>Радиус поражения</p>
+<section class="gadjets">
```
заголовок для секции

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+    <p>Радиус поражения</p>
+<section class="gadjets">
+  <ul>
+    <li class="navigation-item"><img src="#" alt="Очки для Виртуальной реальности"> Виртуальная реальность</li>
```
лучше картинки в декоративную графику

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+  </ul>
+</section>
+
+<section>
```
заголовок секции

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+
+<section>
+  <ul>
+    <li class="navigation-item"><a href="" class="navigation-link">Доставка</a></li>
+    <li class="navigation-item"><a href="" class="navigation-link">Гарантия</a></li>
+    <li class="navigation-item"><a href="" class="navigation-link">Кредит</a></li>
```
это кнопки, которые переключают содержимое справа

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+    <li class="navigation-item"><a href="" class="navigation-link">Гарантия</a></li>
+    <li class="navigation-item"><a href="" class="navigation-link">Кредит</a></li>
+  </ul>
+  <img src="" alt="Черная палочка-перегородка">
```
img не нужен

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+  <img src="" alt="Черная палочка-перегородка">
+</section>
+
+<p>Доставка</p>
```
нужно подобрать другой тег

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+
+<p>Доставка</p>
+<p>Мы с удовольствием доставим ваш товар прямо к вашему подъезду совершенно бесплатно! Ведь мы неплохо заработаем, поднимая его на ваш этаж.</p>
+  <img src="#" alt="Человек-доставщик">
```
не нужно в разметке

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+<p>Мы с удовольствием доставим ваш товар прямо к вашему подъезду совершенно бесплатно! Ведь мы неплохо заработаем, поднимая его на ваш этаж.</p>
+  <img src="#" alt="Человек-доставщик">
+
+  <section>
+    <img src="#" alt="Грузовик">
+    <p>Доставим редкий товар под заказ</p>
+    <img src="#" alt="Восклицательный знак">
+  </section>
```
лишние img, это все одна кнопка для показа модалки, не нужно выносить ее в отделбную секцию

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+      <li class="navigation-item">СДИК</li>
+      <li class="navigation-item">ТоксБерри</li>
+    </ul>
+    <p><a href="#">Подробнее о нас</a></p>
```
параграф не нужен

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+    <p>Вы можете забрать товар сами, заехав в наш офис.Заодно сможете проверить работоспособность покупки. </p>
+      <h3>Интернет-магазин «Девайс»</h3>
+      <p>Санкт-Петербург, набережная реки Карповки, 5, литера П.</p>
+      <p>+7 (812) 812-12-12</p>
```
ссылка

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+      <h3>Время работы</h3>
+      <p>пн—пт: с 10:00 до 22:00<br>
+        сб—вс: с 10:00 до 19:00</p>
+        <p><a href="#">Напишите нам</a></p>
```
параграф не нужен

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+  <section>
+    <h2>Подпишитесь на рассылку</h2>
+    <p>Еженедельный дайджест с обзорами новинок, длительными тест-драйвами, распаковками, а также акции, спецпредложения и вот это вот всё.</p>
+      <label for="newsletter-email"></label>
+      <input type="email" placeholder="Ваш e-mail" id="newsletter-email">
+      <button type="button">Подписаться</button>
+      <img src="#" alt="Значек Ок">
+      <p>Это бесплатно</p>
```
это форма, img не нужен

Serega560 replied:
Обернул в форму , img удалил

Kartaviy-arteom commented on index.html:
```diff
@@ -2,11 +2,140 @@
+  </section>
+</main>
+
+<footer class="page-footer">
```
футер аналогично каталогу

Serega560 replied:
Здесь не понял
у меня на двух страницах footer одинаковый. я с index.html  скопировал в catalog.html

Kartaviy-arteom commented on style.css:
```diff
@@ -0,0 +1,17 @@
+.page-header{
```
будут в дирректории styles

Serega560 replied:
ок, это так, попробовал

Kartaviy-arteom commented on style.css:
```diff
@@ -0,0 +1,17 @@
+.page-header{
+height: 300px;
+left: 140px;
+top: 40px;
+border-radius: 0;
+
+  background: #FFE17F;
+}
+.main-nav{
+font-family: 'Raleway';
+font-style: normal;
+font-weight: 800;
+font-size: 18px;
+line-height: 21px;
+letter-spacing: 0.2em;
+text-transform: uppercase;
+}
```
необходимо соблюдать форматирование

Serega560 replied:
да, понял, это просто проба была

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,192 @@
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
этот тег тут не обязателен, в первом курсе нет адаптивов

Serega560 replied:
понял, я просто с помощью !+Tab делаю эту начинку, там metы уже сразу вписаны

Serega560 committed:
Update index.html

исправил форматирование страниц
вынес каталог, доставка, гарантия, контакты в nav
Обернул в форму Поиск по сайту и кнопку "Найти"
Удалил декоративную графику
удалил img в списках

Serega560 committed:
Update catalog.html

внес изменения

Serega560 committed:
Update index.html

изменил форму
 обернул в span кнопки "Кнопка вперед" и "Кнопка назад
переставил классы в тэгах сразу после тэгов

Serega560 committed:
Доработки

переставил классы в тэгах сразу после тэгов

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,228 @@
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
этот тег на первом курсе, не нужен, но это и не ошибка

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,228 @@
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
```
все стили будут храниться в дирректории styles, не в корне проекта

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,228 @@
+
+        <nav class="navigation">
+          <a href="#">Каталог товаров</a>
+          <button class="dropdown" type="button">Кнопка</button>
```
это все один элемент. тут не нужно два отдельных элемента

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,228 @@
+              <ul class="catolog-filter-list">
+                <li class="catolog-filter-item">
+                  <label class="control">
+                  <input type="checkbox" name="black" checked>Черный
```
label должен быть связан с input и далее. Чекбоксы нужно связать между собой через name

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,228 @@
+            </fieldset>
+          </form>
+
+          <button class="link" type="button" >Показать</button>
```
кнопка внутри формы

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,228 @@
+        </a>
+        <ul>
+          <li class="product-card">
+            <a href="#" class="product-card-link">
```
ссылкой все содержимое карточки лучше тогда

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,228 @@
+          </li>
+        </ul>
+        <div class="">
+          <a href="">Загрузить еще</a>
```
это кнопка

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,228 @@
+      <img class="footer-contacts-logo" src="#" alt="Магазин Девайс" >
+        <address class="cotacts-address">
+          <p class="contacts-address-text">Санкт-Петербург, набережная реки Карповки, 5, литера П.</p>
+          <a class="contacts-phone" href="#" >Тел.: +7 (812) 812-12-12</a>
```
телефон не в этой части футера

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,182 @@
+
+    <main class="main-index">
+      <button type="button">
+        <span class="visiually-hidden">Кнопка вперед</span>
```
лучше написать что она делает, то что это кнопка это понятно.

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,182 @@
+    </header>
+
+    <main class="main-index">
+      <button type="button">
```
из этого нужно собрать секцию в которой будет слайдер

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,182 @@
+      <section class="gadjets">
+        <ul>
+          <li class="navigation-item">
+            <img src="#" alt="Очки для Виртуальной реальности"> Виртуальная реальность
```
это обернуть нужно в ссылки

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,182 @@
+      <section>
+        <ul>
+          <li class="navigation-item">
+            <a class="navigation-link link" href="#">Доставка</a>
```
тут будут кнопки

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,182 @@
+        поднимая его на ваш этаж.</p>
+      </section>
+
+        <p>Доставим редкий товар под заказ</p>
```
это кнопка для модалки

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,182 @@
+        <form action="post">
+          <label for="newsletter-email"></label>
+        <input type="email" placeholder="Ваш e-mail" id="newsletter-email">
+        <button type="button">Подписаться</button>
```
тип у кнопки

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,182 @@
+        <input type="email" placeholder="Ваш e-mail" id="newsletter-email">
+        <button type="button">Подписаться</button>
+        </form>
+        <p>Это бесплатно</p>
```
лучше не p

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,182 @@
+      <img class="footer-contacts-logo" src="#" alt="Магазин Девайс" >
+        <address class="cotacts-address">
+          <p class="contacts-address-text">Санкт-Петербург, набережная реки Карповки, 5, литера П.</p>
+          <a class="contacts-phone" href="#" >Тел.: +7 (812) 812-12-12</a>
```
в href ссылка на телефон

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,182 @@
+        </ul>
 
+        <a href="https://t.me/htmlacademy">
+          <img src="#" alt="Теллеграмм">
```
img эти не нужны

Kartaviy-arteom commented on style.css:
```diff
@@ -0,0 +1,17 @@
+
+  background: #FFE17F;
+}
+.main-nav{
+font-family: 'Raleway';
+font-style: normal;
+font-weight: 800;
+font-size: 18px;
+line-height: 21px;
+letter-spacing: 0.2em;
+text-transform: uppercase;
+}
```
форматирование

Serega560 committed:
Create teleg.svg

Serega560 committed:
Create Экшн.svg

Serega560 committed:
Create Умные.svg

Serega560 committed:
Create Моно.svg

Serega560 committed:
Create грузовик .svg

Serega560 committed:
Create Фит.svg

Serega560 committed:
Create Квадро.svg

Serega560 committed:
Create Палка.svg

Serega560 committed:
Create Вирт.svg

Serega560 committed:
Create you.svg

Serega560 committed:
Create Vector.svg

Serega560 committed:
Create twit.svg

Serega560 committed:
Create man.svg

Serega560 committed:
Create html-academy.svg

Serega560 committed:
Create Ellipse.svg

Serega560 committed:
Create device-yellow.svg

Serega560 committed:
Create Квадракоптер.png

Serega560 committed:
Create product-2-3 (1).jpg

Serega560 committed:
Create product-2-2 (1).jpg

Serega560 committed:
Create product-2-1 (1).jpg

Serega560 committed:
Create product-2 (1).jpg

Serega560 committed:
Create sravnenie.svg

Serega560 committed:
Create search.svg

Serega560 committed:
Create man.svg

Serega560 committed:
Create logo.svg

Serega560 committed:
Create korzina.svg

Serega560 committed:
Create fon -footer.svg

Serega560 committed:
Create button+.svg

Serega560 committed:
Create 01.svg

Serega560 committed:
Create logo-yellowsvg.svg

Serega560 committed:
логотип

добавил логотипы в разметку с адресом на главную страницу
добавил файлы img

Serega560 committed:
Лого +якорь

добавил логотипы
сделал ссылку якорь

Serega560 committed:
Update index.html

изменил чекбоксы
связал label с inputom

Serega560 committed:
Update catalog.html

изменил значения кнопок
объединил чекбоксы

Serega560 committed:
Create Квадрокоптер.png

Serega560 committed:
Delete Квадракоптер.png

Serega560 committed:
Create website.svg

Serega560 committed:
Update catalog.html

добавил контентные изображения и логотипы в хэдере и футэре

добавил им alt  и ширину с высотой

Serega560 committed:
img

добавил контентные изображения и логотипы в хэдере и футэре

добавил им alt  и ширину с высотой

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,226 @@
+        <img class="logo logo-header" src="images/header/logo.svg" width="145" height="36" alt="Логотип Магазина Девайс"/>
+      </a>
+      <form action="https://echo.htmlacademy.ru/courses" method="get">
+        <label for="search"></label>
```
пояснение

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,226 @@
+        </ul>
+      </nav>
+
+      <ul class="navigation-list">
```
перед nav

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,226 @@
+      </form>
+
+      <nav class="navigation">
+        <button class="dropdown" type="button">Каталог товаров</button>
```
ссылка

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,226 @@
+      </ul>
+
+      <section>
+        <h2 class="filter">Фильтр</h2>
```
каталог с фильтрами

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,226 @@
+
+      <section>
+        <h2 class="filter">Фильтр</h2>
+        <h3 class="catalog-filter-title">Стоимость</h3>
```
будет легендой внутри fieldset

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,226 @@
+        <h2 class="filter">Фильтр</h2>
+        <h3 class="catalog-filter-title">Стоимость</h3>
+        0<input class="min" type="range" min="0" max="10000" step="100"/>10000
+        <input class="max" type="range" min="0" max="10000" step="100"/>
```
внутри формы

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,226 @@
+              </li>
+              <li class="catolog-filter-item">
+                <label class="control" for="color-white">
+                  <input type="checkbox" name="color" value="white" id="color-white" checked/>Белый
```
форматирование

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,226 @@
+      </section>
+
+      <section class="select-control">
+        <h2 class="filter">Сортировка</h2>
```
результаты поиска

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,226 @@
+
+      <section class="select-control">
+        <h2 class="filter">Сортировка</h2>
+        <option>Популярные</option>
```
select

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,226 @@
+        <option>Популярные</option>
+      </section>
+      <a class="button button-up" href="#">
+        <span class="visually-hidden">Вверх</span>
```
кнопки

Kartaviy-arteom commented on catalog.html:
```diff
@@ -0,0 +1,226 @@
+            <img src="images/main/jpg/product-2 (1).jpg" width="360" height="380" alt="Любительская селфи-палка"/>
+            <h3 class="product-card-title">Любительская селфи-палка</h3>
+          </a>
+          <b class="product-card-price">500 P</b>
```
цену внутрь ссылки

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,202 @@
+        <span class="visiually-hidden">Кнопка назад</span>
+      </button>
+      <img src="images/main/png/Квадрокоптер.png" width="560" height="560" alt="Квадрокоптер"/>
+      <img src="images/header/01.svg" width="250" height="135" alt="цифры 01" style="background-color: rgb(222, 237, 18)"/>
```
не нужно

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,202 @@
+      <button type="button">
+        <span class="visiually-hidden">Кнопка назад</span>
+      </button>
+      <img src="images/main/png/Квадрокоптер.png" width="560" height="560" alt="Квадрокоптер"/>
```
левая половина изображение и две кнопки

Kartaviy-arteom commented on index.html:
```diff
@@ -1,12 +1,202 @@
+          Еженедельный дайджест с обзорами новинок, длительными тест-драйвами,
+          распаковками, а также акции, спецпредложения и вот это вот всё.
+        </p>
+        <form action="post">
```
метод, post - это метод

Serega560 committed:
Create slider-1.png

Serega560 committed:
Create logo-footer.svg

Serega560 committed:
Create link-all.svg

Serega560 committed:
Delete logo-yellowsvg.svg

Serega560 committed:
Delete website.svg

Serega560 committed:
Delete button+.svg

Serega560 committed:
Create basket.svg

Serega560 committed:
Create button.svg

Serega560 committed:
Create comparison.svg

Serega560 committed:
Delete korzina.svg

Serega560 committed:
Delete sravnenie.svg

Serega560 committed:
Delete product-2 (1).jpg

Serega560 committed:
Delete product-2-1 (1).jpg

Serega560 committed:
Create product-2-1.jpg

Serega560 committed:
Delete product-2-2 (1).jpg

Serega560 committed:
Create product-2-2.jpg

Serega560 committed:
Delete fon -footer.svg

Serega560 committed:
Delete 01.svg

Serega560 committed:
Create product-2-3.jpg

Serega560 committed:
Create product-2.jpg

Serega560 committed:
Delete product-2-3 (1).jpg

Serega560 committed:
Create action-camers.svg

Serega560 committed:
Delete device-yellow.svg

Serega560 committed:
Create fitness bracelets.svg

Serega560 committed:
Delete Ellipse.svg

Serega560 committed:
Delete Квадрокоптер.png

Serega560 committed:
Create logo-footer.svg

Serega560 committed:
Create monopods.svg

Serega560 committed:
Create popular.svg

Serega560 committed:
Create quadrocopters.svg

Serega560 committed:
Create smart-watch.svg

Serega560 committed:
Delete teleg.svg

Serega560 committed:
Create telegram.svg

Serega560 committed:
Create truck.svg

Serega560 committed:
Delete twit.svg

Serega560 committed:
Create twitter.svg

Serega560 committed:
Update Vector.svg

Serega560 committed:
Create virtual-reality.svg

Serega560 committed:
Delete you.svg

Serega560 committed:
Create youtube.svg

Serega560 committed:
Delete Вирт.svg

Serega560 committed:
Delete грузовик .svg

Serega560 committed:
Delete Квадро.svg

Serega560 committed:
Delete Моно.svg

Serega560 committed:
Delete Палка.svg

Serega560 committed:
Delete Умные.svg

Serega560 committed:
Delete Фит.svg

Serega560 committed:
Delete Экшн.svg

Serega560 committed:
Update catalog.html

назвал label
удалил h1  в headere
поменял местами навигацию и список ссылок
добавил fieldset

Serega560 committed:
Update index.html

назвал label
поменял местами навигацию и список ссылок
удалил img 01

Serega560 committed:
Create .gitattributes copy

Serega560 committed:
Delete style.css

Serega560 committed:
Update catalog.html

разбил footer на дивы

Serega560 committed:
Update catalog.html

добавил  styles

Serega560 committed:
Create styles.css

создал styles

Kartaviy-arteom commented on styles/styles.css:
```diff
@@ -0,0 +1,3 @@
+header {
+  background-color: #FFE17F
+}
```
в конце правила будем ставить везде ;
`background-color: #FFE17F;`
